<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="3" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-11-15T03:08:44Z" name="RestAPI Test Automation Test Suite" finished-at="2019-11-15T03:08:47Z" duration-ms="3168">
    <groups>
    </groups>
    <test started-at="2019-11-15T03:08:44Z" name="RestAPI Test Cases" finished-at="2019-11-15T03:08:47Z" duration-ms="3168">
      <class name="com.qa.tests.UpdateAPITest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.UpdateAPITest@25b485ba]" started-at="2019-11-14T21:08:45Z" name="setUp" finished-at="2019-11-14T21:08:45Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="updateAPITest()[pri:0, instance:com.qa.tests.UpdateAPITest@25b485ba]" started-at="2019-11-14T21:08:45Z" name="updateAPITest" finished-at="2019-11-14T21:08:46Z" duration-ms="750" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateAPITest -->
      </class> <!-- com.qa.tests.UpdateAPITest -->
      <class name="com.qa.tests.PostAPITest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.PostAPITest@1f1c7bf6]" started-at="2019-11-14T21:08:44Z" name="setUp" finished-at="2019-11-14T21:08:44Z" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="postAPITest()[pri:0, instance:com.qa.tests.PostAPITest@1f1c7bf6]" started-at="2019-11-14T21:08:44Z" name="postAPITest" finished-at="2019-11-14T21:08:45Z" duration-ms="1248" status="FAIL">
          <exception class="org.apache.http.client.ClientProtocolException">
            <full-stacktrace>
              <![CDATA[org.apache.http.client.ClientProtocolException
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:187)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.qa.client.RestClient.post(RestClient.java:63)
	at com.qa.tests.PostAPITest.postAPITest(PostAPITest.java:65)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.apache.http.ProtocolException: Target host is not specified
	at org.apache.http.impl.conn.DefaultRoutePlanner.determineRoute(DefaultRoutePlanner.java:71)
	at org.apache.http.impl.client.InternalHttpClient.determineRoute(InternalHttpClient.java:125)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.http.client.ClientProtocolException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postAPITest -->
      </class> <!-- com.qa.tests.PostAPITest -->
      <class name="com.qa.tests.DeleteAPITest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.DeleteAPITest@2b546384]" started-at="2019-11-14T21:08:46Z" name="setUp" finished-at="2019-11-14T21:08:46Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="getAPITestWithoutHeaders()[pri:1, instance:com.qa.tests.DeleteAPITest@2b546384]" started-at="2019-11-14T21:08:46Z" name="getAPITestWithoutHeaders" finished-at="2019-11-14T21:08:47Z" duration-ms="672" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAPITestWithoutHeaders -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.DeleteAPITest@2b546384]" started-at="2019-11-14T21:08:47Z" name="setUp" finished-at="2019-11-14T21:08:47Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="deleteAPITestWithHeaders()[pri:2, instance:com.qa.tests.DeleteAPITest@2b546384]" started-at="2019-11-14T21:08:47Z" name="deleteAPITestWithHeaders" finished-at="2019-11-14T21:08:47Z" duration-ms="409" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteAPITestWithHeaders -->
      </class> <!-- com.qa.tests.DeleteAPITest -->
      <class name="com.qa.tests.GetAPITest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.GetAPITest@75c072cb]" started-at="2019-11-14T21:08:46Z" name="setUp" finished-at="2019-11-14T21:08:46Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="getAPITestWithoutHeaders()[pri:1, instance:com.qa.tests.GetAPITest@75c072cb]" started-at="2019-11-14T21:08:46Z" name="getAPITestWithoutHeaders" finished-at="2019-11-14T21:08:46Z" duration-ms="4" status="FAIL">
          <exception class="org.apache.http.client.ClientProtocolException">
            <full-stacktrace>
              <![CDATA[org.apache.http.client.ClientProtocolException
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:187)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.qa.client.RestClient.get(RestClient.java:26)
	at com.qa.tests.GetAPITest.getAPITestWithoutHeaders(GetAPITest.java:45)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.apache.http.ProtocolException: Target host is not specified
	at org.apache.http.impl.conn.DefaultRoutePlanner.determineRoute(DefaultRoutePlanner.java:71)
	at org.apache.http.impl.client.InternalHttpClient.determineRoute(InternalHttpClient.java:125)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.http.client.ClientProtocolException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAPITestWithoutHeaders -->
        <test-method signature="getAPITestWithHeaders()[pri:2, instance:com.qa.tests.GetAPITest@75c072cb]" started-at="2019-11-14T21:08:47Z" name="getAPITestWithHeaders" finished-at="2019-11-14T21:08:47Z" duration-ms="6" status="FAIL">
          <exception class="org.apache.http.client.ClientProtocolException">
            <full-stacktrace>
              <![CDATA[org.apache.http.client.ClientProtocolException
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:187)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
	at com.qa.client.RestClient.get(RestClient.java:40)
	at com.qa.tests.GetAPITest.getAPITestWithHeaders(GetAPITest.java:108)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.apache.http.ProtocolException: Target host is not specified
	at org.apache.http.impl.conn.DefaultRoutePlanner.determineRoute(DefaultRoutePlanner.java:71)
	at org.apache.http.impl.client.InternalHttpClient.determineRoute(InternalHttpClient.java:125)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.http.client.ClientProtocolException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getAPITestWithHeaders -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.qa.tests.GetAPITest@75c072cb]" started-at="2019-11-14T21:08:47Z" name="setUp" finished-at="2019-11-14T21:08:47Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
      </class> <!-- com.qa.tests.GetAPITest -->
    </test> <!-- RestAPI Test Cases -->
  </suite> <!-- RestAPI Test Automation Test Suite -->
</testng-results>
